beacon_node/http_api/src/publish_blocks.rs

```diff
@@ -41,26 +41,40 @@ pub async fn publish_block<T: BeaconChainTypes>(
     };
     let delay = get_block_delay_ms(seen_timestamp, block.message(), &chain.slot_clock);
 
-    debug!(
-        log,
-        "Signed block published to HTTP API";
-        "slot" => block.slot()
-    );
+    let block_root = block_root.unwrap_or_else(|| block.canonical_root());
 
-    // Send the block, regardless of whether or not it is valid. The API
-    // specification is very clear that this is the desired behaviour.
+    let chain_clone = chain.clone();
+    let block_clone = block.clone();
+    let log_clone = log.clone();
+    let sender_clone = network_tx.clone();
 
-    let message = PubsubMessage::BeaconBlock(block.clone());
-    crate::publish_pubsub_message(network_tx, message)?;
+    let publish_fn = move || {
+        debug!(
```

Comment:
I'd been using this log to figure out the earliest point at which the BN saw the block, which comes in handy when troubleshooting latent VC/BN comms. So it might be nice to move this log back where it was and add another one.

Reply:
moved this log back and added a couple more logs here https://github.com/sigp/lighthouse/pull/4168/commits/8f38805f0579470afff9649b0af0a9b973c35883